modules/custom/az_core/src/Plugin/ConfigProvider/QuickstartConfigProvider.php

```diff
@@ -262,14 +262,20 @@ protected function canOverride($name) {
     $lister = \Drupal::service('config_update.config_list');
     /** @var \Drupal\config_update\ConfigDiffer $differ */
     $differ = \Drupal::service('config_update.config_diff');
-
     // Read active config value for name.
     $active = $this->getActiveStorages()->read($name);
     // Find out which module owns the configuration and load the snapshot value.
     $owner = $lister->getConfigProvider($name);
     if (!empty($owner[1])) {
       $snapshot_storage = $this->getConfigSnapshotStorage(ConfigSyncSnapshotterInterface::CONFIG_SNAPSHOT_SET, $owner[0], $owner[1]);
       $snap = $snapshot_storage->read($name);
+      // StorageInterface::read() returns FALSE if the config does not exist.
+      // In this case, we can assume that the configuration is not customized
+      // because it is not present in the snapshot, likely because the module
+      // is newly installed.
+      if ($snap === FALSE) {
+        return TRUE;
+      }
```

Comment:
I've heard of an `isNew` property on config objects before in this ticket: https://www.drupal.org/project/config_sync/issues/2996039

I wonder if that could be used here.

Reply:
https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Config%21StorableConfigBase.php/function/StorableConfigBase%3A%3AisNew/11.x